wiki/Silence/en.md

```diff
@@ -1,43 +1,62 @@
-- Edit profile details (e.g. avatar, "me!" section, etc.)
-- Submit and update beatmaps
-- Partake in [multiplayer](/wiki/Client/Interface/Multiplayer) games
-
-In addition to that, all messages of a silenced user are removed from all chat channels and private conversations.[^chat-cleanup]
+- Use the online [chat](/wiki/Client/Interface/Chat_console) (public and private), both in-game and on the website.
+- Post on any of the osu! forums.
+- Post comments anywhere on the website (beatmaps, changelog, news).
+- Take part in [beatmap discussions](/wiki/Beatmap_discussion).
+- Edit profile details, change the avatar, cover and userpage.
+- Submit and update beatmaps.
+- Partake in [multiplayer](/wiki/Client/Interface/Multiplayer) games.
 
 ::: Infobox
 ![](img/silence-ingame.png "Silence warning in the chat")
 :::
 
-After a user is silenced, an alert will be displayed in place of the chat input field, saying when the silence wears off. The silence record can also be observed at the bottom of the profile on the website, from which it will disappear after 28 days.
+Once a silence is issued to a user, all of their messages in chat are removed and an alert will be displayed in the place of the chat input field containing the duration of the active silence.[^chat-cleanup]
+
+## Durations
 
-## Appealing a silence
+The duration of a silence starts at 5 minutes and generally doubles for each subsequent offence, up to a maximum of 28 days, but the initial duration may vary depending on the offence committed and the user's prior history. The factors considered when determining the length of a silence include:
 
-If you believe you were silenced unfairly, contact the [account support team](/wiki/People/Account_support_team#accounts@ppy.sh) at [accounts@ppy.sh](mailto:accounts@ppy.sh) to contest the silence. Make sure to do it from the email linked to your osu! account; mention your osu! username, as well as the approximate date and time when the silence was handed out.
+- **Atmosphere:** The current chat environment's state.
+- **Frequency:** Repetitiveness regardless of previous silences given.
+- **History:** Records of previous infringements.
+- **Severity:** The seriousness of the offence committed.
+
+In some cases, users may receive a final warning before an [account restriction](/wiki/Help_centre/Account_restrictions) is placed, providing an opportunity to cease the offending behaviour.
 
 ## Common reasons for silences
 
-*Caution: Engaging in "inappropriate behaviour" in private messages is not against the rules, unless said inappropriate behaviour is being delivered to unwilling or unconsenting recipients. If reported, appropriate punishments will be placed.*
+Some common reasons why users get silenced in public chat include, but are not limited to:
+
+- **Spamming or flooding:** Self-explanatory.
+- **Caps abuse:** Chatting with all uppercase letters.
+- **Inappropriate behavior/conversation:** The public chat is not a place for such discussions.
```

Comment:
```suggestion
- **Inappropriate behaviour/conversation:** The public chat is not a place for such discussions.
```

Reply:
IMO there should be a brief description of what discussions are being referred to here.
I would think something like this:
```suggestion
- **Inappropriate behavior/conversation:** The public chat is not a place for discussions not appropriate for all ages, or for heavily inflammatory topics.
```

Reply:
i would think any inflammatory topic is not welcome so maybe the `heavily` is not needed

Reply:
@chromb thoughts on this? asking cuz of https://github.com/ppy/osu-wiki/pull/12505#discussion_r1866769530